### ISSUE TITLE
Document SOCKS forwarding

### ISSUE THREAD

**Issue by clue** [clue/psocksd#12]
We need to document the SOCKS forwarding (`via` command). Some diagrams would certainly help.

Refs #10.

**Comment by phrazer** [comment]
agree :+1:

**Comment by clue** [comment]
PRs welcome :)

```
$ php bin/psocksd 
use direct connection to target
SOCKS proxy server listening on localhost:9050

via
no command arguments given. Valid options are:
    list
        list all entries
    default <target>
        set given <target> socks proxy as default target
    reject <host>
        reject connections to the given host
    add <host> <target> [<priority>]
        add new <target> socks proxy for connections to given <host>
    remove <entryId>
        emove entry with given <id> (see "list")
    reset
        clear and reset everything and only connect locally

via add youtube.com reject
reject

via add :80 direct
use direct connection to target

via add :443 10.20.30.40:9050                        
use socks://10.20.30.40:9050 as next hop (resolve remotely)

via list
Id:  Host:        Port:  Prio:  Target:
1    youtube.com  *      0      -reject-
2                 443    0      socks://10.20.30.40:9050
3                 80     0      -direct-
0    *            *      100    -direct-

```

**Comment by clue** [comment]
Closed via #30
